README.md

```diff
@@ -1 +1,54 @@
-# TodoList
\ No newline at end of file
+# ToDo List App
+
+Todo List is a web application that allows users to add/remove the list of "to-dos". It provides an interface for adding to do activities, for visualizing the list of added tasks to complete. This is achieved using plain HTML, CSS, JavaScript objects, arrays, DOM, basic events, and webpack.
+
+
+![Screenshot](src/project_creenshot.png)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Git flow
+- GitHub
+- web pack
+
+### Prerequisites
+
+- Terminal (Bash)
+- Git
+- JavaScript
+  
+### Setup: How to start the Todo_List app
+
+- git clone the repo: `https://github.com/Chike1990/ToDo_List`
+- cd into: `Todo list folder`
+- npm install
+- npm start
```

Comment:
- [ ] Kindly add the start script in your package.json. Also don't forget to add the dependencies needed.It should look like this 
```
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "start": "webpack serve --open"
  },
```